concrete/src/Command/Task/Input/Definition/Field.php

```diff
@@ -65,11 +71,20 @@ public function getDescription(): string
         return $this->description;
     }
 
+    /**
+     * @return string|null
```

Comment:
I don't think we need this phpdoc: what's returned by the method is already fully declared by `: ?string`